src/components/LibraryUnit.tsx

```diff
@@ -36,19 +36,24 @@ export const LibraryUnit = ({
     if (!elementsToRender) {
       return;
     }
-    const svg = exportToSvg(elementsToRender, {
-      exportBackground: false,
-      viewBackgroundColor: oc.white,
-    });
-    for (const child of ref.current!.children) {
-      if (child.tagName !== "svg") {
-        continue;
+    let svg: SVGSVGElement;
+    const current = ref.current!;
+
+    const getSvg = async () => {
+      svg = await exportToSvg(elementsToRender, {
+        exportBackground: false,
+        viewBackgroundColor: oc.white,
+      });
+      for (const child of ref.current!.children) {
+        if (child.tagName !== "svg") {
+          continue;
+        }
+        ref.current!.removeChild(child);
       }
-      ref.current!.removeChild(child);
-    }
-    ref.current!.appendChild(svg);
+      ref.current!.appendChild(svg);
+    };
 
-    const current = ref.current!;
+    getSvg();
     return () => {
       current.removeChild(svg);
```

Comment:
prob fine, but if wanted to guarantee this won't throw, we should wrap this call in in `svg` null check.